docs/docsite/rst/porting_guides/porting_guide_2.7.rst

```diff
@@ -103,6 +103,15 @@ hashing algorithm being used with this filter.
 Deprecated
 ==========
 
+Expedited Deprecation: Removal of the params module option in ``ldap_attr`` and ``ldap_entry``
+----------------------------------------------------------------------------------------------
+
+The ``params`` module option in ``ldap_attr`` and ``ldap_entry`` are deprecated on a short cycle (to
```

Comment:
Hi @s-hertel is the deprecation scheduled to occur with 2.10 or sooner? If its happening with 2.10 I think it would provide better clarity to rephrase the first sentence to  "The ``params`` module option in ``ldap_attr`` and ``ldap_entry`` is deprecated in Ansible 2.10 due to circumventing Ansible's normal option handling.

Reply:
@cbudz the options are deprecated now, and will be gone in Ansible 2.10.

Reply:
thanks @felixfontein, please disregard my comment in that case.